lua/octo/util.lua

```diff
@@ -209,8 +209,10 @@ function M.update_issue_metadata(bufnr)
   local comments = api.nvim_buf_get_var(bufnr, "comments")
   for i, m in ipairs(comments) do
     metadata = m
-    mark = api.nvim_buf_get_extmark_by_id(bufnr, constants.OCTO_COMMENT_NS, metadata.extmark, {details = true})
-    start_line, end_line, text = M.get_extmark_region(bufnr, mark)
+    header_mark = api.nvim_buf_get_extmark_by_id(bufnr, constants.OCTO_HEADER_NS, metadata.header_mark, {details = true})
+    body_mark = api.nvim_buf_get_extmark_by_id(bufnr, constants.OCTO_COMMENT_NS, metadata.extmark, {details = true})
+    header_line = header_mark[1]
+    start_line, end_line, text = M.get_extmark_region(bufnr, body_mark)
```

Comment:
This shouldnt be needed in the new approach

BTW: I cant use `:wq` on this comment floats. I have to do `:w` and then `q`

Reply:
Closing in regards of [this comment](https://github.com/pwntester/octo.nvim/pull/116#issuecomment-789578727).

PS: Let's remember this issue. But I think this should get resolved automatically with the new approach.